INSTALL.md

```diff
@@ -841,6 +841,18 @@ Don't perform FIPS module Power On Self Tests.
 This option MUST be used for debugging only as it makes the FIPS provider
 non-compliant. It is useful when setting breakpoints in FIPS algorithms.
 
+### enable-fips-jitter
+
+Use the CPU Jitter library as a FIPS validated entropy source.
+
+This option will only produce a compliant FIPS provider if you have:
+
+1. independently performed the required [SP 800-90B] entropy assessments;
+2. met the minimum required entropy as specified by [jitterentropy-library] and
+3. have had the resulting FIPS provider certified by the [CMVP].
+
+Failure to do all of these will produce a non-compliant FIPS provider.
```

Comment:
This is imprecise

```
Step 3 is Obtain ESV certificate for jitterentropy-library [ESV]
Step 4 have had the resulting FIPS provider certified by the [CMVP]

[ESV] https://csrc.nist.gov/Projects/cryptographic-module-validation-program/entropy-validations
```

Note that the same build/ESV can be used multiple times for different CMVP submissions of the cryptographic module, whilst compatible the same ESV cert can be reused over and over again.